src/client/graphics/layers/EventsDisplay.ts

```diff
@@ -187,6 +191,33 @@ export class EventsDisplay extends LitElement implements Layer {
     });
   }
 
+  onDisplayChatEvent(event: DisplayChatMessageUpdate) {
+    const myPlayer = this.game.playerByClientID(this.clientID);
+    if (
+      event.playerID != null &&
```

Comment:
Since chat should never have a null playerID (unlike message):
```suggestion
      event.playerID === null ||
```